modules/axtask/src/lib.rs

```diff
@@ -48,6 +48,21 @@ cfg_if::cfg_if! {
         mod task;
         mod api;
         mod wait_queue;
+        #[cfg(feature = "irq")]
+        /// load average
+        pub mod loadavg;
+        // TODO: if irq is disabled, what value should AVENRUN be?
+        static mut AVENRUN: [u64; 3] = [0, 0, 0];
```

Comment:
What is `AVENRUN` short for?

Reply:
Maybe "average run queue". I copied it from linux kernel code directly